src/scss/components/tabset/_index.scss

```diff
@@ -124,7 +124,7 @@ hx-tabset {
 hx-tabset:not([hx-defined]) {
   hx-tab:first-of-type {
     background-color: $gray-0;
-    background-color: var(--hxTabset-hxTab-first-of-type-bgcolor, $gray-0);
+    background-color: var(--hxTabset-hxTab-first-of-type-bgcolor, $gray-0);//there is no token for bg color
```

Comment:
Please update to use design tokens. If lower level definition missing, use `$rax-color-gray-0`.

Keep the comment and add a space after the semicolon.

Reply:
Changes have been done, please review.